app/scripts/pages/board/boardController.js

```diff
@@ -1,4 +1,23 @@
-app.controller('boardController', ['$scope', function($scope) {
+app.controller('boardController', boardController);
+
+boardController.$inject = ['$scope', '$routeParams', 'dashboardService', 'listService'];
+
+function boardController($scope, $routeParams, dashboardService, listService) {
+  const boardId = $routeParams.boardId;
+
+  $scope.boards = [];
+
+  dashboardService.getDashboards().then(
```

Comment:
Please, use a special method to get only the current board.

Reply:
https://trello-api-by-angie.herokuapp.com/doc/#Dashboard
(/dashboards/:dashboardId Get a dashboard by it's key)